PROCESSORS.md

```diff
@@ -313,7 +313,7 @@ In the list below, the names of required properties appear in bold. Any other pr
 
 ### Description
 
-DefragmentText splits and merges incoming flowfiles so cohesive messages are not split between them
+DefragmentText splits and merges incoming flowfiles so cohesive messages are not split between them, it can handle multiple inputs differentiated by the <em>absolute.path</em> flow file attribute.
```

Comment:
Nitpicking, but I would split this into two sentences:
```suggestion
DefragmentText splits and merges incoming flowfiles so cohesive messages are not split between them. It can handle multiple inputs differentiated by the <em>absolute.path</em> flow file attribute.
```